new-hunt-setup.md

```diff
@@ -80,5 +82,6 @@ The authorized users for a Small Board deployment are the Google users who have
 
 Some other things you may want to tweak before hunt include:
 
+* Setting a `DJANGO_SECRET_KEY` config var so that user sessions will not expire after each restart. You can generate a key using `python -c "import secrets; print(secrets.token_urlsafe())"`.
 * Setting `DEBUG=False`
```

Comment:
Should we move these up to the "Heroku config variables" section above? They should probably be considered non-optional.

Reply:
Good suggestion. I removed this section entirely and integrated it with the above sections.